products/dns/api.yaml

```diff
@@ -14,7 +14,10 @@
 --- !ruby/object:Api::Product
 name: Google Cloud DNS
 prefix: gdns
-base_url: https://www.googleapis.com/dns/v1/
+versions:
+  - !ruby/object:Api::Product::Version
+    name: v1
+    base_url: https://www.googleapis.com/dns/v1/
```

Comment:
Shouldn't you put `default=true`? With the current logic, the compiler should have thrown an exception...

Reply:
No longer necessary.